src/test/kotlin/fi/metatavu/pakkasmarja/services/erp/test/functional/sap/SapMock.kt

```diff
@@ -0,0 +1,102 @@
+package fi.metatavu.pakkasmarja.services.erp.test.functional.sap
+
+import fi.metatavu.pakkasmarja.services.erp.test.sap.mock.apis.EntriesApi
+import fi.metatavu.pakkasmarja.services.erp.test.sap.mock.models.Entry
+import org.eclipse.microprofile.config.ConfigProvider
+import java.io.InputStreamReader
+
+/**
+ * SAP mock client
+ */
+class SapMock: AutoCloseable {
+
+    private val sapMockUrl: String
+        get() {
+            val config = ConfigProvider.getConfig()
+            return config.getValue("fi.metatavu.pakkasmarja.odata-mock-url", String::class.java)
+        }
+
+    override fun close() {
+        EntriesApi(basePath = sapMockUrl).deleteEntries(name = null)
+    }
+
+    /**
+     * Mock SAP items
+     *
+     * @param ids item ids
+     */
+    fun mockItems(vararg ids: String) {
+        ids.forEach(this::mockItem)
+    }
+
+    /**
+     * Mock single SAP item
+     *
+     * @param id item id
+     */
+    private fun mockItem(id: String) {
```

Comment:
Move private methods after public (and protected)

Reply:
Fixed